Model/Transaction.php

```diff
@@ -141,6 +141,31 @@ public function getTransactionOrder()
         return $oOrder;
     }
 
+    /**
+     * Returns transaction's payment method name.
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    public function getTransactionPaymentMethodName()
+    {
+        if ($this->getPaymentType()) { // there is an order associated with the transaction
+            $oPayment = PaymentMethodHelper::getPaymentById($this->getPaymentType());
+            return $oPayment->oxpayments__oxdesc->value;
+        }
+
+        $oXml = simplexml_load_string($this->getResponseXML());
+        $sPaymentId = $oXml->{'payment-methods'}->{'payment-method'}['name'];
+        $oPayment = PaymentMethodHelper::getPaymentById('wd' . $sPaymentId);
+        // payment with id from xml response exists in database
```

Comment:
Please add an **empty line** before the comment to make it more readable.

Reply:
👍